com.unity.multiplayer.mlapi/Tests/Editor/SnapshotRttTests.cs

```diff
@@ -0,0 +1,33 @@
+using MLAPI;
+using NUnit.Framework;
+
+namespace MLAPI.EditorTests
+{
+    public class SnapshotRttTests
+    {
+        [Test]
+        public void TestRtt()
+        {
+            var snapshot = new SnapshotSystem();
```

Comment:
How about 
- a test that does > k_RingSize stores?
- a test that ack's something that wasn't sent?

Reply:
yeah, I'd rename this test to be something that communicates "send more, receive less".
also vice-versa option would be "a test that ack's something that wasn't sent?" aka "send less, receive more"

I'd also add:
- send 2x more than `k_RingSize`
- randomization: out-of-order send/recv variations (if ooo is ever possible, IDK)
- randomization: sending/receiving (notify/ack) intentionally with varying artificatial latencies (WaitForSeconds)

at least, these are the test cases comes to my mind in a minute — perhaps more tests if applicable/sensible?